ledger/sandbox/src/main/scala/com/digitalasset/platform/index/StandaloneIndexerServer.scala

```diff
@@ -10,54 +10,63 @@ import com.digitalasset.platform.common.logging.NamedLoggerFactory
 import com.digitalasset.platform.common.util.{DirectExecutionContext => DEC}
 import com.digitalasset.platform.index.config.{Config, StartupMode}
 
-import scala.concurrent.{Await, Future, Promise}
 import scala.concurrent.duration._
+import scala.concurrent.{Await, ExecutionContext, Future, Promise}
+import scala.util.control.NonFatal
 
 // Main entry point to start an indexer server.
 // See v2.ReferenceServer for the usage
 object StandaloneIndexerServer {
+  private val asyncTolerance: FiniteDuration = 10.seconds
+
+  private implicit val executionContext: ExecutionContext = DEC
```

Comment:
As you are no longer referencing `DEC` multiple times it no longer makes sense to import `DirectExecutionContext` aliased.

Reply:
Good point. I just imported `DirectExecutionContext.implicitEC`.